source/agora/common/TransactionPool.d

```diff
@@ -162,7 +162,7 @@ public class TransactionPool
     public void remove (const ref Transaction tx) @trusted
     {
         // delete inputs of transaction from the set of Input hashes
-        foreach (ref input; tx.inputs)
+        foreach (const ref input; tx.inputs)
```

Comment:
So technically this should be in the second commit right ?

Reply:
Correct, I'll rebase it :)